src/feature.js

```diff
@@ -20,98 +20,25 @@ class Feature {
-        break;
-      case "FB2L2":
-        browser.prefs.setIntPref("privacy.fastblock.list", 2);
-        browser.prefs.setIntPref("browser.fastblock.timeout", 2000);
-        browser.prefs.setBoolPref("browser.fastblock.enabled", true);
-        browser.prefs.setBoolPref("privacy.trackingprotection.enabled", false);
-        browser.prefs.setBoolPref("network.http.tailing.enabled", false);
-        break;
-      case "FB2L3":
-        browser.prefs.setIntPref("privacy.fastblock.list", 3);
-        browser.prefs.setIntPref("browser.fastblock.timeout", 2000);
-        browser.prefs.setBoolPref("browser.fastblock.enabled", true);
-        browser.prefs.setBoolPref("privacy.trackingprotection.enabled", false);
-        browser.prefs.setBoolPref("network.http.tailing.enabled", false);
-        break;
-      case "FB5L0":
-        browser.prefs.setIntPref("privacy.fastblock.list", 0);
-        browser.prefs.setIntPref("browser.fastblock.timeout", 5000);
-        browser.prefs.setBoolPref("browser.fastblock.enabled", true);
-        browser.prefs.setBoolPref("privacy.trackingprotection.enabled", false);
-        browser.prefs.setBoolPref("network.http.tailing.enabled", false);
-        break;
-      case "FB5L1":
-        browser.prefs.setIntPref("privacy.fastblock.list", 1);
-        browser.prefs.setIntPref("browser.fastblock.timeout", 5000);
-        browser.prefs.setBoolPref("browser.fastblock.enabled", true);
-        browser.prefs.setBoolPref("privacy.trackingprotection.enabled", false);
-        browser.prefs.setBoolPref("network.http.tailing.enabled", false);
-        break;
-      case "FB5L2":
-        browser.prefs.setIntPref("privacy.fastblock.list", 2);
-        browser.prefs.setIntPref("browser.fastblock.timeout", 5000);
-        browser.prefs.setBoolPref("browser.fastblock.enabled", true);
-        browser.prefs.setBoolPref("privacy.trackingprotection.enabled", false);
-        browser.prefs.setBoolPref("network.http.tailing.enabled", false);
-        break;
-      case "FB5L3":
-        browser.prefs.setIntPref("privacy.fastblock.list", 3);
-        browser.prefs.setIntPref("browser.fastblock.timeout", 5000);
-        browser.prefs.setBoolPref("browser.fastblock.enabled", true);
-        browser.prefs.setBoolPref("privacy.trackingprotection.enabled", false);
-        browser.prefs.setBoolPref("network.http.tailing.enabled", false);
-        break;
-      case "Control":
-        browser.prefs.setBoolPref("browser.fastblock.enabled", false);
-        break;
-      case "TT":
-        browser.prefs.setIntPref("privacy.fastblock.list", 0);
-        browser.prefs.setBoolPref("browser.fastblock.enabled", true);
-        browser.prefs.setBoolPref("privacy.trackingprotection.enabled", false);
-        browser.prefs.setBoolPref("network.http.tailing.enabled", true);
-        break;
+    variation = VARIATIONS[variation.name];
+
+    for (const pref in variation.prefs) {
```

Comment:
The idea was to have Normandy recruit users who only have default values for the relevant prefs, so I don't think this is a concern for us in the configure stage. I think what we need to look after is if the user changes a pref after already enrolled

Reply:
turns out Normandy will look after that too - "by the next session" so we will only need to take care of this if it is not quick enough